### PR TITLE
BlinkDownloaderB.py

### PR DESCRIPTION
**Pull request by JSLSlim**
Blink video downloader.  Updated 2019-05-30.  Adds additional console output, logging.  Creates different folder structure.

### CODE REVIEW

**Comment by JohnMurga** [comment]
Nice stuff in there, however, does not work for me as I don't have D: drive ...
Also I suspect that despite the use of os.sep it'll do some funky things on Linux.

Also not sure why you had to delete the script that I am using in the merge request ?

**Comment by Zipsnet** [comment]
A solutions would be , adding something like this I giuess ..

from sys import platform as _platform

if _platform == "linux" or _platform == "linux2":
   # linux
elif _platform == "darwin":
   # MAC OS X
elif _platform == "win32":
   # Windows
elif _platform == "win64":
    # Windows 64-bit

**Comment by JSLSlim** [comment]
John,

 

My apologies for the deletion.  I am new to GitHub and don’t know the interface very well yet and didn’t know this would cause any problems.

 

I do not know much about the Linus OS/Filesystem but as far as I know the Python os.sep call is OS agnostic and relies on the interpreter to return the proper symbol.  

 

The header on the code should have some basic instructions on editing the variables so that you can make the code work for your individual system, but once again, I don’t have any specific knowledge on how to address a Linux filesystem.

 

Thanks for the reply,

John

**Comment by JohnMurga** [comment]
No problem, os.sep would work fine, it is just that on Linux it would probably create a file in a relative path with "d:" in the name which would be a bit weird. The way to avoid this could be to either pass in a path as a parameter (which is OK as the script already requires credentials), or to make it write relative to where it is run from.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+As forked from MattTW's master page:
```
Comment (MattTW) [inline README.md]: Does not need merged

**Pull request closed without merging**